tests/scripts/remote_monitoring_tests/rest_apis/test_list_recommendations.py

```diff
@@ -801,16 +811,17 @@ def test_list_recommendations_with_only_latest(latest, cluster_type):
         data = response.json()
         assert response.status_code == SUCCESS_STATUS_CODE
         assert data[0]['experiment_name'] == experiment_name
-        assert data[0]['kubernetes_objects'][0]['containers'][0]['recommendations']['notifications']['112101'][
-                   'message'] == 'Cost Recommendations Available'
+        assert data[0]['kubernetes_objects'][0]['containers'][0]['recommendations']['notifications'][
+                   NOTIFICATION_CODE_FOR_RECOMMENDATIONS_AVAILABLE][
+                   'message'] == RECOMMENDATIONS_AVAILABLE
     experiment_name = None
     response = list_recommendations(experiment_name, latest)
 
     list_reco_json = response.json()
     assert response.status_code == SUCCESS_200_STATUS_CODE
 
     # Validate the json against the json schema
-    errorMsg = validate_list_reco_json(list_reco_json, list_reco_json_schema)
+    errorMsg = validate_list_reco_json(list_reco_json, medium_term_list_reco_json_schema)
```

Comment:
This should be reco_json_schema.  Have you tested the changes?

Reply:
Updated. 
Also, Ran the complete functional tests, didn't find any issues in that.